src/etc/checkstyle-suppressions.xml

```diff
@@ -22,6 +22,6 @@
 
 <suppressions>
     <suppress
-            checks="JavadocStyleCheck|JavadocMethodCheck|JavadocTypeCheck|JavadocVariableCheck|LineLength|FinalParameters|AvoidStarImport"
+            checks="JavadocStyleCheck|JavadocMethodCheck|JavadocTypeCheck|JavadocVariableCheck|LineLength|FinalParameters|AvoidStarImport|AvoidInlineConditionals"
```

Comment:
Damn, I thought you activated the AvoidInlineConditionals rule. I'm actually perfectly fine with it deactivated. Can't read correctly (or think correctly). My mind must be on something else ...

That all good for me ! :+1: